### ISSUE TITLE
Send function to nameof.full

### ISSUE THREAD

**Issue by syxov** [dsherret/ts-nameof#17]
It's possible to send function to the `nameof`. How about the same functionality for `nameof.full`?
Something like
```typescript
declare namespace nameof {
    function full<T>(func?: (obj: T) => void, periodIndex?: number): string;
}
```

**Comment by dsherret** [comment]
Yup, would `nameof.full<MyInterface>(i => i.prop.otherProp);` outputting `"prop.otherProp"` make sense?

**Comment by syxov** [comment]
Yes, it's what I want, thanks

**Comment by dsherret** [comment]
@syxov implemented in 0.10.0. Please let me know if you run into any problems or it doesn't work as expected.
